apis/apiextensions/v1/composition_transforms.go

```diff
@@ -250,6 +253,11 @@ func stringConvertTransform(input interface{}, t *StringConversionType) (interfa
 		return strings.ToUpper(str), nil
 	case ConversionTypeToLower:
 		return strings.ToLower(str), nil
+	case ConversionTypeToBase64:
+		return base64.StdEncoding.EncodeToString([]byte(str)), nil
+	case ConversionTypeFromBase64:
+		s, err := base64.StdEncoding.DecodeString(str)
+		return string(s), err
```

Comment:
Can you `errors.Wrap` this error with some context (an error string constant) please - something like `errors.Wrap(err, errDecodeString)`.

Reply:
Done